src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -68,4 +70,32 @@ public static boolean isNonZeroUnsignedInteger(String s) {
             return false;
         }
     }
+
+    /**
+     * Returns a partially {@return encoded URL} from a {@params s}
+     */
+    public static String partiallyEncode(String s) {
+        requireNonNull(s);
+        return s.replaceAll(" ", "+")
+                .replaceAll(",", "%2C")
+                .replaceAll("#", "%23");
+    }
+
+    //@@author A0143832J
+    /**
+     * check if a string contains another substring, ignoring case
+     * @param str String to be checked
+     * @param searchStr Substring
+     * @return true if {@code str} contains {@code searchStr}, false otherwise
+     */
+    public static boolean containsIgnoreCase(String str, String searchStr) {
```

Comment:
Isn't there a similar functionality in the String class? As far as possible, don't re-invent the wheel. Re-use code that's already given.